tool/planet/etcd.go

```diff
@@ -747,28 +746,24 @@ func convertError(err error) error {
 	return err
 }
 
-// systemctl runs a local systemctl command in non-blocking mode.
-// TODO(knisbet): I'm using systemctl here, because using go-systemd and dbus appears to be unreliable, with
-// masking unit files not working. Ideally, this will use dbus at some point in the future.
-func systemctl(ctx context.Context, operation, service string) error {
-	return systemctlCmd(ctx, operation, service, "--no-block")
+func disableService(ctx context.Context, service string) error {
```

Comment:
```suggestion
func disableService(ctx context.Context, service string) error {
	err := systemd.DisableService(ctx, service)
	if err != nil {
		return trace.Wrap(err)
	}
	return trace.Wrap(waitForEtcdStopped(ctx))
}
```
This is `lib/utils/systemd#DisableService` (bar the call to `waitForEtcdStopped`)